web/libs/ui/src/lib/button/button.tsx

```diff
@@ -0,0 +1,98 @@
+import { cn } from "../../utils/utils";
+import type { ButtonHTMLAttributes, PropsWithChildren } from "react";
+import styles from "./button.module.scss";
+
+const variants = {
+  primary: styles["variant-primary"],
+  neutral: styles["variant-neutral"],
+  negative: styles["variant-negative"],
+  positive: styles["variant-positive"],
+  warning: styles["variant-warning"],
+  inverted: styles["variant-neutral-interted"],
+};
+
+const looks = {
+  filled: styles["look-filled"],
+  string: styles["look-string"],
+  outlined: styles["look-outlined"],
+};
+
+const sizes = {
+  default: styles["size-default"],
+  compact: styles["size-compact"],
+  comfortable: styles["size-comfortable"],
+};
```

Comment:
```
const sizes = {
  default: styles["size-default"],
  compact: styles["size-compact"],
  comfortable: styles["size-comfortable"],
};
```
I don't see any sizes defined in the css

Reply:
updated. initially there were questions for @ricardoantoniocm, resolved and sizes are supported now